### PR TITLE
Add PICKLE token metadata

### PR DESCRIPTION
**Pull request by larrythecucumber321**
# Add PICKLE token metadata

ERC-20: https://etherscan.io/address/0x429881672B9AE42b8EbA0E26cD9C73711b891Ca5
NEP-141: https://explorer.near.org/accounts/429881672b9ae42b8eba0e26cd9c73711b891ca5.factory.bridge.near

### CODE REVIEW

File: tokens/pickle.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
....442-2.406 6.298-2.314 8.532.203m-32.046 31.252c4.622 2.359 5.07 6.725 1.19 11.617-3.156 3.981-4.144 8.581-2.767 12.889.648 2.028 1.209 2.991 3.437 5.898 3.539 4.617 2.418 9.034-2.637 10.396-3.805 1.024-7.546-1.174-10.251-6.024-.421-.755-.859-1.462-.973-1.572-.933-.897-2.265-5.359-2.769-9.279-.459-3.568.045-10.452.827-11.301.192-.209.351-.659.352-1 .003-1.632 3.713-7.675 6.06-9.873 2.794-2.618 4.881-3.103 7.531-1.751m30.371 33.347c3.589 1.701 4.424 5.906 1.878 9.451-3.582 4.988-4.941 12.549-2.913 16.206.367.66.667 1.391.669 1.625s.322.773.714 1.2 1.234 1.464 1.872 2.306c3.734 4.934-1.823 12.383-7.228 9.687-1.687-.842-4.956-4.149-4.956-5.015 0-.221-.149-.403-.332-.403-.499 0-3.001-5.256-4.013-8.428-.59-1.851-.59-11.292 0-13.144 3.243-10.17 9.285-15.865 14.309-13.485m-30.371 32.653c4.622 2.359 5.07 6.725 1.19 11.617-3.156 3.981-4.144 8.581-2.767 12.889.65 2.036 1.217 3.005 3.447 5.898 3.393 4.401 2.399 9.051-2.265 10.601-3.843 1.276-7.711-.99-10.633-6.229-.421-.755-.855-1.462-.965-1.572-.45-.45-1.6-3.277-1.6-3.934 0-.388-.158-.877-.35-1.086-1.252-1.359-1.252-14.201 0-15.56.192-.209.351-.659.352-1 .003-1.632 3.713-7.675 6.06-9.873 2.794-2.618 4.881-3.103 7.531-1.751" fill="#0c540c" />
```
Comment (sept-en) [inline tokens/pickle.svg:2]: The icon is super-big, this won't work, unfortunately. Please try reducing it to 5-7 KB.

File: tokens/pickle.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "ethereum_address": "0x429881672B9AE42b8EbA0E26cD9C73711b891Ca5",
+    "name": "PickleToken",
+    "symbol": "PICKLE",
+    "decimals": 18,
+    "reference": "",
+    "reference_hash": ""
+}
```
Comment (sept-en) [inline tokens/pickle.json:8]: Please add "icon" field (even if it's empty if you can't convert it) and "aurora_address" - we deployed a contract for you on `0x291c8fceaca3342b29cc36171deb98106f712c66`

**Review by sept-en — CHANGES_REQUESTED** [review]
Thanks for PR! Added some comments need to be addressed before merging.

**Comment by larrythecucumber321** [comment]
Thank you for the feedback @sept-en! Hopefully this does the job 🙏

**Review by sept-en — APPROVED** [review]

**Pull request merged**